### DISCUSSION TITLE
Web interface of Sherlock

### DISCUSSION THREAD

**Discussion by Thaldos** [sherlock-project/sherlock#1235] [General]
Hello guys,

I love your project and would just to say great job guys !

This message is just to told you that I just created a web interface in php / html of your project : https://findaccountsbyusername.com/
All my project is open source and free https://github.com/Thaldos/findaccountsbyusername
And I also created an open and free API : https://findaccountsbyusername.com/api.php
See more details on the main page to use it.

I apologize about the time to response, my server is a little bit slim (1Go of RAM).

Bye

**Comment by Mjt66** [comment]
First link, not correct info, ,2nd & 3rd both give error code not working..thanks though..let me know if you get working

**Reply by Thaldos** [reply]
> ,2nd give error code not working

Fixed
3th is normal, you have to send a POST request to this url, as explained on https://findaccountsbyusername.com/

**Comment by jpmortaza** [comment]
Very good.
